packages/material-ui/src/ListItem/ListItem.js

```diff
@@ -304,12 +333,14 @@ ListItem.propTypes /* remove-proptypes */ = {
    * If `true`, the list item is focused during the first mount.
    * Focus will also be triggered if the value changes from false to true.
    * @default false
+   * @deprecated will be removed in v6, checkout `ListItemButton` instead
```

Comment:
I guess the initial intent of Jun was so that we don't forget to remove deal logic in the next BC window opportunity. Maybe if we add a runtime deprecating message with `deprecatedPropType()`, then it would start to be valuable?

Reply:
Good point.